docs/rules/string-buffer-to-builder.md

```diff
@@ -54,5 +54,69 @@ __Post__
 StringBuilder localStringBuffer = new StringBuilder();
 ```
 
+### Bytecode JDK 1.8 (Optional)
```

Comment:
What do you indicate by writing __(Optional)__?

Reply:
I forgot to remove it, as I just took the structure from the rule-template.md

Reply:
Ok, please change that.